views/layout.html

```diff
@@ -76,7 +76,7 @@
       </div>
       <div class="flex flex-column ml5">
       <p class="ttu ma0">News</p>
-        <a href="/publications" class="be-white">Latest Publications</a>
+        <a href="/publications1" class="be-white">Latest Publications</a>
```

Comment:
It looks a bit weird to have a `/publications1` url. Was this intentional?

Reply:
Thanks for catching that!